docs/release-notes/NuGet-6.7.md

```diff
@@ -0,0 +1,175 @@
+---
+title: NuGet 6.7 Release Notes
+description: Release notes for NuGet 6.7 including new features, bug fixes, and DCRs.
+author: jeffkl
+ms.author: jeffkl
+ms.date: 7/31/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.7 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.7**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.7](https://visualstudio.microsoft.com/downloads/) | [7.0.400](https://dotnet.microsoft.com/download/dotnet-core/7.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.7
+
+* NuGetAudit should warn when no vulnerability data available - [#12610](https://github.com/NuGet/Home/issues/12610)
+
+* remove period from NU190X - [#12600](https://github.com/NuGet/Home/issues/12600)
+
+* Package Source Mapping status for selected package in Details Pane - [#12586](https://github.com/NuGet/Home/issues/12586)
+
+* Add VulnerabilityInfo APIs into NuGet.Protocol - [#12518](https://github.com/NuGet/Home/issues/12518)
+
+* Signing:  raise actionable message on Linux if verification results in untrusted failure - [#12459](https://github.com/NuGet/Home/issues/12459)
+
+* Warn when vulnerabilities are detected during PackageReference restore  - [#12289](https://github.com/NuGet/Home/issues/12289)
```

Comment:
NuGet Audit was disabled in 6.7

Reply:
@zivkan Should 6.8 be set as the milestone on that issue?